include/log.h

```diff
@@ -28,6 +32,28 @@
 	fprintf(stderr, __VA_ARGS__);                                \
 	fprintf(stderr, ANSI_COLOR_RESET);
 
+#define pr_debug_fd(...)                                     \
+	if (get_verbose()) {                                 \
+		pr_debug(__VA_ARGS__);                       \
```

Comment:
Let's just do this here, to avoid checking for verbosity two times.
```c
	if (get_verbose()) {                               \
		fprintf(stdout, "VERBOSE:%s\t", __func__); \
		fprintf(stdout, __VA_ARGS__);              \
	}   \
```